### PR TITLE
Completed about_dictionaries koan

### PR DESCRIPTION
**Pull request by stevendes**
Completed and ready for review!

### CODE REVIEW

File: python2/koans/about_dictionaries.py [hunk 1]
```diff
@@ -13,47 +13,47 @@ def test_creating_dictionaries(self):
...
             ('red warrior', 'green elf', 'blue valkyrie', 'yellow dwarf',
              'confused looking zebra'),
             42)
 
-        self.assertEqual(__, len(cards))
-        self.assertEqual(__, cards['green elf'])
-        self.assertEqual(__, cards['yellow dwarf'])
+        self.assertEqual(5, len(cards))
+        self.assertEqual(42, cards['green elf'])
+        self.assertEqual(42, cards['yellow dwarf'])
```
Comment (glpuga) [inline python2/koans/about_dictionaries.py:59]: nit: line break before end of file.

File: python2/koans/about_dictionaries.py [hunk 2]
```diff
@@ -13,47 +13,47 @@ def test_creating_dictionaries(self):
...
-        self.assertEqual(__, len(babel_fish.values()))
-        self.assertEqual(__, 'one' in babel_fish.keys())
-        self.assertEqual(__, 'two' in babel_fish.values())
-        self.assertEqual(__, 'uno' in babel_fish.keys())
-        self.assertEqual(__, 'dos' in babel_fish.values())
+        self.assertEqual(2, len(babel_fish.keys()))
+        self.assertEqual(2, len(babel_fish.values()))
+        self.assertEqual(True, 'one' in babel_fish.keys())
+        self.assertEqual(False, 'two' in babel_fish.values())
+        self.assertEqual(False, 'uno' in babel_fish.keys())
```
Comment (glpuga) [inline python2/koans/about_dictionaries.py:48]: Q: What type are the return values of `values()` and `keys()`?

File: python2/koans/about_dictionaries.py [hunk 3]
```diff
@@ -13,47 +13,47 @@ def test_creating_dictionaries(self):
...
         babel_fish = {'one': 'uno', 'two': 'dos'}
-        self.assertEqual(__, len(babel_fish))
+        self.assertEqual(2, len(babel_fish))
 
     def test_accessing_dictionaries(self):
         babel_fish = {'one': 'uno', 'two': 'dos'}
-        self.assertEqual(__, babel_fish['one'])
-        self.assertEqual(__, babel_fish['two'])
+        self.assertEqual('uno', babel_fish['one'])
+        self.assertEqual('dos', babel_fish['two'])
```
Comment (glpuga) [inline python2/koans/about_dictionaries.py:27]: Q1: Can `None` be a valid key?

Comment (glpuga) [inline python2/koans/about_dictionaries.py:27]: Q2: Can `None` be a valid value?

Comment (glpuga) [inline python2/koans/about_dictionaries.py:27]: May sound like an Idle question, but `None` is precisely the return value of any function if you forget to call `return` on it...

**Review by glpuga — APPROVED** [review]
LGTM

File: python2/koans/about_dictionaries.py (hunk 2 above)
Comment (stevendes) [reply python2/koans/about_dictionaries.py:48]: It returns `list` for both.

File: python2/koans/about_dictionaries.py (hunk 3 above)
Comment (stevendes) [reply python2/koans/about_dictionaries.py:27]: Yes, it can.

Comment (stevendes) [reply python2/koans/about_dictionaries.py:27]: Also can.

**Pull request merged**